employee.rb

```diff
@@ -0,0 +1,44 @@
+class Employee
+  attr_accessor :salary, :name
+
+  def initialize(name, salary)
+    @name = name
+    @salary = salary
+  end
+
+  def give_raise(amount)
+    @salary = (@salary.to_i + amount).to_s
+  end
+
+  def review(number)
```

Comment:
I see... you're automatically generating a review based on their score.  That's a nice bonus touch.